Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>plugins {\r\n    id 'fabric-loom' version '1.7.3'\r\n    id 'maven-publish'\r\n}\r\n\r\nversion = '0.0.1'\r\ngroup = 'github.sonicunnamedmod'\r\n\r\nrepositories {\r\n    mavenCentral()\r\n    maven { url = 'https://maven.fabricmc.net/' }\r\n}\r\n\r\ndependencies {\r\n    minecraft \"com.mojang:minecraft:1.21.1\"\r\n    mappings \"net.fabricmc:yarn:1.21.1+build.1:v2\"\r\n    modImplementation \"net.fabricmc:fabric-loader:0.15.11\"\r\n\r\n    modImplementation \"net.fabricmc.fabric-api:fabric-api:0.102.0+1.21.1\"\r\n}\r\n\r\njava {\r\n    toolchain {\r\n        languageVersion = JavaLanguageVersion.of(21)\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision f79fe7573f6ba8b09bb388b3862611e6586b9367)
+++ b/build.gradle	(date 1782846057933)
@@ -3,7 +3,7 @@
     id 'maven-publish'
 }
 
-version = '0.0.1'
+version = '0.0.4'
 group = 'github.sonicunnamedmod'
 
 repositories {
